### PR TITLE
Unit Test: Bolt\Boltpay\Helper\Log

### PR DESCRIPTION
**Pull request by johnmccombs1**
# Description
Unit Test: Bolt\Boltpay\Helper\Log 

Fixes: https://app.asana.com/0/564264490825835/1155443326571665

\#changelog

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [x] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [x] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have created or modified unit tests to sufficiently cover my changes.
- [x] I have added my Asana task link and provided a changelog message if applicable.

### CODE REVIEW

File: Helper/Log.php [hunk 1]
```diff
@@ -32,7 +31,7 @@
 class Log extends AbstractHelper
 {
     /**
-     * @var Logger
+     * @var BoltLoger
```
Comment (matt-thomason) [inline Helper/Log.php]: Can you fix the alias to `BoltLogger`?

File: Test/Unit/Helper/LogTest.php [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2019 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Test\Unit\Helper;
+
+use Bolt\Boltpay\Logger\Logger as BoltLoger;
```
Comment (matt-thomason) [inline Test/Unit/Helper/LogTest.php]: Same down here.

Comment (johnmccombs1) [reply Test/Unit/Helper/LogTest.php]: @matt-thomason I updated the code

File: Helper/Log.php (hunk 1 above)
Comment (johnmccombs1) [reply Helper/Log.php]: @matt-thomason I updated the code

**Pull request merged**